pkg/statistics/table.go

```diff
@@ -629,25 +629,37 @@ func (t *Table) GetStatsHealthy() (int64, bool) {
 // The Column should be visible in the table and really has analyzed statistics in the stroage.
 // Also, if the stats has been loaded into the memory, we also don't need to load it.
 // We return the Column together with the checking result, to avoid accessing the map multiple times.
-func (t *Table) ColumnIsLoadNeeded(id int64, fullLoad bool) (*Column, bool) {
+// The first bool is whether we have it in memory. The second bool is whether this column has stats or not.
```

Comment:
I think "has stats" is a bit ambiguous.
```suggestion
// The first bool is whether we have it in memory. The second bool is whether this column has stats in the system table or not.
```